helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -964,6 +964,10 @@ livenessProbe:
 # Specifies whether to apply for k8s cluster with windows worker node.
 isWindows: false
 
+# Openshift SecurityContextConstraints can be overriden in this field.
+# NOTE: This config will only be used when distribution=openshift
```

Comment:
Please mention that the fields will be merged into the default config that can be found at <link to the template>